tests/test_lm/test_integration/test_jsrun/test_2/res_set

```diff
@@ -0,0 +1,2 @@
+cpu_index_using: physical
+rank: 0: { host: 1; cpu: {0}; gpu: {1}}
```

Comment:
Do gpu indices start with 1? Any specific reason it is 'gpu: {1}' and not 'gpu: {0}'?

Reply:
They start by `0`, but what does it matter when verification is based on the number of resources used and not which resources?

Reply:
Correct, we only test for the number of cpus/gpus assigned and not their index. I will start make my initial suggestion.

Two reasons:
* We simply remain consistent between cpu and gpu. Going with the first gpu (as we do for cpu) makes sense than picking a random number. There should be a reason on why we chose a particular configuration to test.
* When/if we add tests to check the index of the cpu/gpu assigned, you will have to go in and change the comments as well as the tests (or you write your test to check if gpu 1 was acquired -- again why gpu 1?).

To me, this is trivial and starting from the first index makes sense. Testing if we got gpu 1 ( or any other cpu/gpu) can be an additional test, of course.

Reply:
FWIW only test 2 tries to assign gpu 1. Rest all assign gpu 0.

Reply:
For consistency, I would also start both CPU and GPU from 0 (or both from 1 that would also be consistent but potentially confusing as it might indicate that the number starts from 1)